security/firely-auth/firely-auth-settings.rst

```diff
@@ -272,6 +273,7 @@ See :ref:`firely_auth_deploy_sqlite` and :ref:`firely_auth_deploy_sql` for detai
 
 - ``Type``: select the type of store to use
 - ``PasswordHashIterations``: number of password hash iterations to prevent brute force attacks. Default 600000. Sync this value when using Firely Auth Management App :ref:`firely_auth_mgmt`.
+- ``LogSqlQueryParameterValues``: when you configured logging of executed queries, the parameter values that are sent to the database are hidden by default. By putting this setting to ``true``, the values will be unhidden and visible in the logs. This might expose sensitive data.
```

Comment:
@rgrotenbreg, do you mean that they would need to adjust logging in the logsettings? I think they would need to enable LogSqlQueryParameterValues together with "Vonk.Repository.Sql.Raw": "Information", to show this query information right? I think it would be good to specify these logsettings a bit, because I don't think we mention this anywhere else.